### PR TITLE
Don't pass change_set if it's edit

### LINKED ISSUE #2727: NameError: uninitialized constant EditChangeSet
Related to https://github.com/pulibrary/figgy/issues/2722

### PR DESCRIPTION
**Pull request by eliotjordan**
Closes #2727

### CODE REVIEW

File: app/assets/javascripts/save_and_ingest_handler.es6 [hunk 1]
```diff
@@ -10,7 +10,7 @@ export default class SaveAndIngestHandler {
       this.button_element.val('Searching...')
       let change_set = window.location.pathname.split('/').pop()
       let qs = {}
-      if (change_set !== 'new') {
+      if (change_set !== 'new' || change_set !== 'edit') {
```
Comment (tpendragon) [inline app/assets/javascripts/save_and_ingest_handler.es6]: Shouldn't this be `&&`?

Comment (eliotjordan) [reply app/assets/javascripts/save_and_ingest_handler.es6]: Totally. Was thinking "and" but typed out "or".

Comment (eliotjordan) [reply app/assets/javascripts/save_and_ingest_handler.es6]: Fixed.

**Comment by eliotjordan** [comment]
@tpendragon Is this acceptable now?

**Comment by tpendragon** [comment]
@eliotjordan Sorry for the delay!

**Pull request merged**